### PR TITLE
Chapter 9 Author Review

### PR DESCRIPTION
**Pull request by szimmer**

### CODE REVIEW

File: 09-ncvs-vignette.Rmd [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
+<!-- author review -->
 
-The NCVS has been ongoing since 1992. An earlier survey, the National Crime Survey, was run from 1972 to 1991 (@ncvs_tech_2016). The survey is administered using a rotating panel. When an address enters the sample, the residents of that address are interviewed every six months for a total of 7 interviews. If the initial residents move away from the address during the period, the new residents are included in the survey, and people are not followed when they move.
+The National Crime Victimization Survey (NCVS) is a household survey sponsored by the Bureau of Justice Statistics (BJS), which collects data on criminal victimization, including characteristics of the crimes, offenders, and victims. Both household and personal crimes include violent and non-violent crimes. The target population of this survey is all people in the United States age 12 and older living in housing units and noninstitutional group quarters.
```
Comment (rpowell22) [inline 09-ncvs-vignette.Rmd:5]: Should probably mention this is the US NCVS or the US BJS just to be extra clear up front.

File: 09-ncvs-vignette.Rmd [hunk 2]
```diff
@@ -1,22 +1,24 @@
...
 
-NCVS data is publicly available and distributed by Inter-university Consortium for Political and Social Research (ICPSR)^[https://www.icpsr.umich.edu/web/ICPSR/series/95], with data going back to 1992. The vignette in this book will include data from 2021 (@ncvs_data_2021). The NCVS data structure is complicated, and the User's Guide contains examples for analysis in SAS, SUDAAN, SPSS, and Stata, but not R (@ncvs_user_guide). This vignette will adapt those examples for R.
+The NCVS has been ongoing since 1992. An earlier survey, the National Crime Survey, was run from 1972 to 1991 (@ncvs_tech_2016). The survey is administered using a rotating panel. When an address enters the sample, the residents of that address are interviewed every six months for a total of 7 interviews. If the initial residents move away from the address during the period, the new residents are included in the survey, and people are not followed when they move.
```
Comment (rpowell22) [inline 09-ncvs-vignette.Rmd:7]: The way these citations are showing up in the parenthesis is wrong.  Need to fix throughout this chapter (maybe elsewhere as well)

File: 09-ncvs-vignette.Rmd [hunk 3]
```diff
@@ -1,22 +1,24 @@
...
+The NCVS has been ongoing since 1992. An earlier survey, the National Crime Survey, was run from 1972 to 1991 (@ncvs_tech_2016). The survey is administered using a rotating panel. When an address enters the sample, the residents of that address are interviewed every six months for a total of 7 interviews. If the initial residents move away from the address during the period, the new residents are included in the survey, and people are not followed when they move. 
+
+NCVS data is publicly available and distributed by Inter-university Consortium for Political and Social Research (ICPSR)^[https://www.icpsr.umich.edu/web/ICPSR/series/95], with data going back to 1992. The vignette in this book will include data from 2021 (@ncvs_data_2021). The NCVS data structure is complicated, and the User's Guide contains examples for analysis in SAS, SUDAAN, SPSS, and Stata, but not R (@ncvs_user_guide). This vignette will adapt those examples for R.
```
Comment (rpowell22) [inline 09-ncvs-vignette.Rmd:9]: At the end of this section, let's add in the research questions or some motivation as to what we're going to be look at.  It takes a while to get to the 4 things later on in the chapter, so would be good to state things clearly up front.

File: 09-ncvs-vignette.Rmd [hunk 4]
```diff
@@ -1,22 +1,24 @@
...
-- Incident Record - `YEARQ`, `IDHH`, `IDPER`
-- 2021 Collection Year Incident - `YEARQ`, `IDHH`, `IDPER`
+  - Address Record - `YEARQ`, `IDHH`
+  - Household Record - `YEARQ`, `IDHH`
+  - Person Record - `YEARQ`, `IDHH`, `IDPER`
+  - Incident Record - `YEARQ`, `IDHH`, `IDPER`
+  - 2021 Collection Year Incident - `YEARQ`, `IDHH`, `IDPER`
 
-We will focus on the household, person, and incident files. From these files, we selected a subset of columns for examples and have included the subset on the book's GitHub repository, and the file subsets will be used in this vignette. Find the dataset in the book's GitHub repository or download the complete files at ICPSR^[https://www.icpsr.umich.edu/web/NACJD/studies/38429].
+We will focus on the household, person, and incident files. From these files, we selected a subset of columns for examples and have included the subset on the book's GitHub repository, and the file subsets will be used in this vignette. Find the dataset in the book's GitHub repository or download the complete files at ICPSR^[https://www.icpsr.umich.edu/web/NACJD/studies/38429].
```
Comment (rpowell22) [inline 09-ncvs-vignette.Rmd:21]: No longer on github, update or remove.

File: 09-ncvs-vignette.Rmd [hunk 5]
```diff
@@ -28,7 +30,7 @@ To calculate victimization rates, we need to know the victimization status for b
 
 Each record on the incident file represents one victimization, which is not the same as one incident. Some victimizations have several instances that make it difficult for the victim to differentiate the details of these incidents, labeled as series crimes. Appendix A of the User's Guide indicates how to calculate the series weight in other statistical languages. That code is adapted here for R. Essentially, if a victimization is a series crime, its series weight is top-coded at 10 based on the number of actual victimizations. If an incident is a series crime, but the number of occurrences is unknown, the series weight is set to 6. A description of the variables used in this chunk is included in the following table:
 
-|Variable|Description|Value|Label|
+|Variable|Description|Value|Label|
```
Comment (rpowell22) [inline 09-ncvs-vignette.Rmd:33]: FYI the table is off the page in PDF.

File: 09-ncvs-vignette.Rmd [hunk 6]
```diff
@@ -102,7 +104,7 @@ In the following example, we will create the following indicators:
 
 A brief codebook for this section is as follows:
 
-|Variable|Description|Value|Label|
+|Variable|Description|Value|Label|
```
Comment (rpowell22) [inline 09-ncvs-vignette.Rmd:107]: FYI the table is off the page in PDF.

File: 09-ncvs-vignette.Rmd [hunk 7]
```diff
@@ -205,7 +207,7 @@ inc_ind %>%
         AAST_Other)
 ```
 
-In the code above, some checking of derived variables is done. This is a good point to pause to look at the output of crosswalks between an original variable and a derived to check the logic was programmed correctly and everything ends up in the bucket you expect. 
+In the code above, some checking of derived variables is done. This is a good point to pause to look at the output of crosswalks between an original variable and a derived to check the logic was programmed correctly and everything ends up in the bucket you expect.
```
Comment (rpowell22) [inline 09-ncvs-vignette.Rmd:210]: Move this above the code where you do this.  There is a LOT of code to skim through without knowing what it is that you are doing/why.

File: 09-ncvs-vignette.Rmd [hunk 8]
```diff
@@ -315,7 +317,7 @@ Brief codebook for household variables:
 
 Brief codebook for person variables:
 
-|Variable|Description|Value|Label|
+|Variable|Description|Value|Label|
```
Comment (rpowell22) [inline 09-ncvs-vignette.Rmd:320]: FYI the table is off the page in PDF.

File: 09-ncvs-vignette.Rmd [hunk 9]
```diff
@@ -530,9 +532,9 @@ We will discuss four estimate types as follows:
 
 $$ \hat{t}_D = \sum_{ijkl \in D} v_{ijkl}$$
 
-where $v_{ijkl}$ is the series-adjusted victimization weight for household $i$, respondent $j$, reporting period $k$, and victimization $l$, that is `WGTVICCY`.
+where $v_{ijkl}$ is the series-adjusted victimization weight for household $i$, respondent $j$, reporting period $k$, and victimization $l$, that is `WGTVICCY`.
```
Comment (rpowell22) [inline 09-ncvs-vignette.Rmd:535]: these second parts are not indented like the rest of the text for each numbered item.  Can these be intended to line up?

File: 09-ncvs-vignette.Rmd [hunk 10]
```diff
@@ -530,9 +532,9 @@ We will discuss four estimate types as follows:
 
 $$ \hat{t}_D = \sum_{ijkl \in D} v_{ijkl}$$
 
-where $v_{ijkl}$ is the series-adjusted victimization weight for household $i$, respondent $j$, reporting period $k$, and victimization $l$, that is `WGTVICCY`.
+where $v_{ijkl}$ is the series-adjusted victimization weight for household $i$, respondent $j$, reporting period $k$, and victimization $l$, that is `WGTVICCY`. 
 
-2. Victimization proportions estimate characteristics among victimizations or victims. Victimization proportions are calculated using the incident design object. The estimated victimization proportion for domain $D$ across level $a$ of covariate $A$, $\hat{p}_{A_a,D}$ is
+2. Victimization proportions estimate characteristics among victimizations or victims. Victimization proportions are calculated using the incident design object. The estimated victimization proportion for domain $D$ across level $a$ of covariate $A$, $\hat{p}_{A_a,D}$ is
```
Comment (rpowell22) [inline 09-ncvs-vignette.Rmd:537]: What are domains and covariates?

File: 09-ncvs-vignette.Rmd [hunk 11]
```diff
@@ -677,7 +679,7 @@ pers_est_df %>%
     locations = cells_column_spanners(spanners = "viol_span")
   ) %>%
   tab_footnote(footnote = "Excludes persons of Hispanic origin",
-               locations = cells_stub(rows = Level != "Hispanic")) %>%
+               locations = cells_stub(rows = Level != "Hispanic")) %>%
```
Comment (rpowell22) [inline 09-ncvs-vignette.Rmd:682]: This is putting a reference on ALL other labels in the table and not just all other labels in the Race section.  May need to call out the specific race/ethnicity items here.

**Review by rpowell22 — CHANGES_REQUESTED** [review]

File: 09-ncvs-vignette.Rmd (hunk 2 above)
Comment (ivelasq) [reply 09-ncvs-vignette.Rmd:7]: @rpowell22 How should they look? I see "(Bureau of Justice Statistics (2017))".

Comment (ivelasq) [reply 09-ncvs-vignette.Rmd:7]: Never mind, think I got it!

**Pull request merged**